backend/core/gemini.py

```
@@ -20,7 +20,7 @@ async def generate_text(prompt: str) -> str:
     return response.text
 
 
-async def generate_structured_output(prompt: str, response_model: type[T]) -> T:
+async def generate_structured_outputt[T](prompt: str, response_model: type[T]) -> T:
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (backend.core.gemini, line 23)'

(E0001)